Source/WebCore/platform/libwpe/PlatformPasteboardLibWPE.cpp

```diff
@@ -59,7 +59,10 @@ void PlatformPasteboard::getTypes(Vector<String>& types) const
 
     for (unsigned i = 0; i < pasteboardTypes.length; ++i) {
         auto& typeString = pasteboardTypes.strings[i];
-        types.append(String({ typeString.data, typeString.length }));
+        if (typeString.length <= std::numeric_limits<size_t>::max())
+            types.append(String({ typeString.data, static_cast<size_t>(typeString.length) }));
+        else
+            types.append(String({ typeString.data, std::numeric_limits<size_t>::max() }));
```

Comment:
Any reason to not use `std::min`? I suppose something like this could work:


```suggestion
        const auto length = std::min(static_cast<size_t>(typeString.length), std::numeric_limits<size_t>::max());
        types.append(String({ typeString.data, length }));
```

Reply:
yes. I prefer your option. Looks better.